src/SFA.DAS.AANHub.Data/Repositories/NotificationsWriteRepository.cs

```diff
@@ -11,5 +11,9 @@ internal class NotificationsWriteRepository : INotificationsWriteRepository
 
     public NotificationsWriteRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
 
-    public void Create(Notification notification) => _aanDataContext.Notifications.Add(notification);
+    public async Task<Guid> Create(Notification notification, CancellationToken cancellationToken)
+    {
+        await _aanDataContext.Notifications.AddAsync(notification, cancellationToken);
```

Comment:
There was no need to use AddAsync as this call does not hit database at all. There are specific use case to use AddAsync which do not apply here.